samtranslator/model/sam_resources.py

```diff
@@ -60,6 +60,16 @@
 MAX_CANARY_UNIQUE_ID_LENGTH = 5
 CANARY_NAME_PREFIX = "sam-"
 
+# The default values for ComparisonOperator, Threshold and Period based on the MetricName provided by the user
+# These default values were acquired from the Create Canary page in the Synthetics Canary dashboard
+DEFAULT_METRIC_VALUES = {
+    "SuccessPercent": {"ComparisonOperator": "LessThanThreshold", "Threshold": 90, "Period": 300},
+    "Failed": {"ComparisonOperator": "GreaterThanOrEqualToThreshold", "Threshold": 1, "Period": 300},
+    "Duration": {"ComparisonOperator": "GreaterThanThreshold", "Threshold": 30000, "Period": 900},
+}
+# the main metrics produced by Synthetics Canary
+VALID_CANARY_METRICS = ["SuccessPercent", "Failed", "Duration"]
```

Comment:
nit: Can reduce duplication by
```suggestion
VALID_CANARY_METRICS = list(DEFAULT_METRIC_VALUES)
```

Reply:
Even though what @aahung suggested works, I would prefer us to be little more explicit so possibly `list(DEFAULT_METRIC_VALUES.keys())`.